client/components/global-notices/README.md

```diff
@@ -1,5 +1,45 @@
 Global Notices
-==========
+==============
+
+Render notices from global application state.
+
+This component should be rendered exactly _one_ time, presumably in the global application layout.
+
+Adding a notice
+---------------
+
+Notices should be added by dispatching a notice action. See [notice actions](../../state/notices/actions.js).
+
+Dispatching a notice from a component might look like this:
+
+```js
+import { successNotices } from 'state/notices/actions';
+
+function MyComponent() {
+  return (
+    <button onClick={ this.props.successNotice( 'Objective achieved!' ) } />Click me!</button>
+  );
+}
+
+export default connect( null, { successNotice } )( MyComponent );
+```
+
+Usage
+-----
+
+It's unlikely you'll need to render GlobalNotices, because that will be handled by layout.
```

Comment:
Instead of `layout`, should we say `<Layout />` to be a bit more specific?

Reply:
This is rendered by both `Layout` and `LayoutLoggedOut`, so the generic "layout" is probably good.